0.2-Elliptic-Curve-Math.ipynb

```diff
@@ -185,8 +175,8 @@
     "\n",
     "Consider:\n",
     "\n",
-    "* `a + b = b + a` over secp256k1\n",
-    "* `a * b = b * a` over secp256k1"
+    "* `a + b == b + a` over secp256k1\n",
```

Comment:
I'm now unsure about the change to double equality.
on one hand as a programmer it's easier to understand. on another hand when it comes as an equation it looks a bit weird.

Reply:
I could go either way on this.